spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'order'
+
+describe Order do
+
+  subject(:order) { described_class.new(menu_klass = menu, order_summary_klass = OrderSummary.new, sms_klass = sms)}
+  let(:sms) { double('sms')}
+  let(:menu) { double('menu', :dishes => { "Golden French Fries" => 0.5 }) }
+
+  context 'selecting an item from the menu' do
+
+    it 'allows a customer to select an item' do
+      expect(order.add_item("Golden French Fries")).to eq "1 Golden French Fries(s) added to your order"
```

Comment:
Metrics/LineLength: Line is too long. [104/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)